galaxy_ng/app/api/ui/serializers/execution_environment.py

```diff
@@ -118,6 +120,22 @@ def get_tags(self, obj):
         return tags
 
 
+class ContainerManifestDetailSerializer(ContainerManifestSerializer):
+    def get_config_blob(self, obj):
+        with obj.config_blob._artifacts.first().file.open() as f:
+            config_json = json.load(f)
+
+        return {
+            'digest': obj.config_blob.digest,
+            'media_type': obj.config_blob.media_type,
+            'data': config_json
+        }
+
+    # class Meta:
+    #     model = super().Meta.model
+    #     fields = super().Meta.fields + ('config_blob')
```

Comment:
this lines aren't needed to include the config_blob field?

Reply:
oops, I forgot to delete that. Thanks